lldb/test/API/lang/swift/array_bridged_enum/bridging-header.h

```diff
@@ -0,0 +1,12 @@
+#ifndef bridging_header_h
+#define bridging_header_h
```

Comment:
do bridging headers need these? Normal objc doesn't need these because `#import` has this behavior.